app-crypt/gpgme/gpgme-1.24.3-r1.ebuild

```diff
@@ -44,26 +44,21 @@ REQUIRED_USE="
 "
 
 # - On each bump, update dep bounds on each version from configure.ac!
-RDEPEND="
+COMMON_DEPEND="
 	>=app-crypt/gnupg-2
 	>=dev-libs/libassuan-2.5.3:=
 	>=dev-libs/libgpg-error-1.46-r1:=
-	cxx? ( !dev-cpp/gpgmepp )
-	python? (
-		!dev-python/gpgmepy
-		${PYTHON_DEPS}
-	)
+	python? ( ${PYTHON_DEPS} )
 	qt5? ( dev-qt/qtcore:5 )
-	qt6? (
-		!dev-libs/qgpgme
-		dev-qt/qtbase:6
-	)
+	qt6? ( dev-qt/qtbase:6 )
 "
-DEPEND="
-	${RDEPEND}
-	test? (
-		qt5? ( dev-qt/qttest:5 )
-	)
+DEPEND="${COMMON_DEPEND}
+	test? ( qt5? ( dev-qt/qttest:5 ) )
+"
+RDEPEND="${COMMON_DEPEND}
+	cxx? ( !>=dev-cpp/gpgmepp-2 )
```

Comment:
Took me a second as I'd forgot our plan ;)

Right, we're using <2 and such as stubs for older versions to depend on.

Reply:
@thesamesam not yet in this PR is a dev-python/gpgmepy metapackage as it is not needed for KDE packages.